src/main/java/Aly.java

```diff
@@ -0,0 +1,242 @@
+import java.util.Scanner;
+
+public class Aly {
+
+    private static final int MAX_TASKS = 100;
+    private static final String LINE_SEPARATOR = "-----------------------------------------------------------------------------";
+    private static final String LOGO = "    _      _     _   _\n"
+            + "   / \\    | |   \\ \\ / /\n"
+            + "  / _ \\   | |    \\ V / \n"
+            + " / ___ \\  | |__   | |  \n"
+            + "/_/   \\_\\ |____|  |_|  \n";
+    private static Scanner in = new Scanner(System.in);
+    private static Task[] tasks = new Task[MAX_TASKS];
+
+    private static void printLine() {
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    private static void initialise() {
+        boolean isExit = false;
```

Comment:
"isExit" is not a grammatically appropriate name for this variable. "exit" (verb) should suffice. Furthermore, the name should describe what it is meant to help exit. Currently, its purpose is ambiguous

Reply:
Boolean variable naming conventions include using auxiliary verbs like “is” to prefix Boolean variables